src/test/java/com/passbook/sparkeighteen/controller/UserControllerTest.java

```diff
@@ -96,4 +96,28 @@ public void validRequest_errorResponse_LoginSuccessfull() throws Exception {
         assertEquals(response.getStatusCode(), HttpStatus.OK);
 
     }
+
+    @Test
+    public void validRequest_successfulResponse_DeleteSuccessfull() throws Exception {
```

Comment:
Naming?

Reply:
is this correct way to name a test: validUserId_successfullResponse_DeleteProfileSuccessfull()